src/CTA.Rules.Actions/ObjectCreationExpressionActions.cs

```diff
@@ -34,5 +36,71 @@ ExpressionSyntax action(SyntaxGenerator syntaxGenerator, ObjectCreationExpressio
             return action;
         }
 
+        public Func<SyntaxGenerator, ObjectCreationExpressionSyntax, ExpressionSyntax> GetReplaceOrAddObjectPropertyIdentifierAction(string oldMember, string newMember, string newValueIfAdding)
+        {
+            Func<SyntaxGenerator, ObjectCreationExpressionSyntax, ExpressionSyntax> action = (SyntaxGenerator syntaxGenerator, ObjectCreationExpressionSyntax node) =>
```

Comment:
This declaration can be simplified. Please run code analysis on the commit to get suggestions from editorconfig.

Reply:
Awesome, thanks Mark. I will always check messages from code analysis from now on. Completed as mentioned.